EIPS/eip-draft_state_rent_H_prepayment.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: <to be assigned>
+title: State Rent H - Fixed Prepayment for accounts
+author: Alexey Akhunov (@AlexeyAkhunov)
+discussions-to: to be added later
```

Comment:
Please insert a forum here, it can be changed later and could be a single thread for "all things state rent" on magicians.